packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -404,6 +407,34 @@ void main() {
     ),
   });
 
+  testUsingContext('plugin project supports web', () async {
```

Comment:
The logic that includes web is kinda weird compares to other platforms.
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/template.dart#L112-L114
As this PR doesn't try to change the logic there, let's add another test to ensure the web related files are not created if web is not requested. WDYT?

Reply:
I added `unexpectedPaths` to the normal plugin creation (where by default `web` is not requested):

```dart
      unexpectedPaths: <String>[
        'lib/flutter_project_web.dart',
      ],
```

Isn't that enough?

Reply:
oh I missed that. that's good enough thanks.